### DISCUSSION TITLE
Any working and maintaned forks of Docz?

### DISCUSSION THREAD

**Discussion by morewings** [pedronauck/docz#1660] [General]
## Question

**Description**
*Any working and maintaned forks of Docz?*

This project is abandoned, but I still like the idea.

**Comment by lucastrvsn** [comment]
Yeah, this project has been abandoned. Maybe I'll move to storybook.

**Comment by pedronauck** [comment]
Hi guys, we're trying to put it back on trails again :)

**Comment by el1s7** [comment]
Back on trails? Why don't you archive it or at least warn users that it's not working

I spent a few hours trying to get this working and it's useless
I could've made a custom documentation with the time spent

**Comment by HonkingGoose** [comment]
@el1s7 and @morewings Have you seen Docusaurus v2? That might fit your usecase for a documentation site. 😉 

Link to Docusaurus project on GitHub: https://github.com/facebook/docusaurus
